Automattic/wp-calypso
# Add `ProductIcon` component to `@automattic/components` package

As proposed in pbtFFM-11-p2, this PR updates and moves the `ProductIcon` component to the `@automattic/components` package.

<img width="799" alt="Screenshot 2019-12-06 at 18 33 59" src="https://user-images.githubusercontent.com/478735/70343219-00a11700-1857-11ea-930c-b0ef1e94ec39.png">

Note that this PR doesn't replace the existing occurrences of the `ProductIcon` that can be found in the `client/components` folder. The adoption of the new component in Calypso (and Jetpack) will be part of a subsequent PRs.

#### Changes proposed in this Pull Request

* Add `ProductIcon` component to the `@automattic/components` package

#### Testing instructions



* Go to https://wpcalypso.wordpress.com/devdocs/design/product-icon
* Confirm that the plans and products icons are rendered correctly.

Fixes n/a
